Ska/engarchive/fetch.py

```diff
@@ -563,6 +597,25 @@ def get_msid_data_from_server(h5_slice, filename):
 
         return (vals, times, bads, colnames)
 
+    @staticmethod
+    def _get_msid_data_from_maude(content, tstart, tstop, msid, unit_system):
+        """
+        Get time and values for an MSID from MAUDE.
+        Returned values are (for now) all assumed to be good.
+        """
+        out = maude.get_msids(msids=msid, start=DateTime(tstart).greta,
```

Comment:
Probably want to wrap this in a try/except since there is a real possibility that this will fail miserably.  In that case it is nice to re-raise with a more informative message for the user explaining that something went wrong with MAUDE.